Used to provide correct OpenLDAP include directory and to remove bogus 
OpenLDAP lib directory, so that pkglint doesn't complain.
Patch was developed in-house; it is Solaris specific and
will not be contributed upstream.

--- python-ldap-2.4.25/setup.cfg.orig	Fri May  6 13:26:21 2016
+++ python-ldap-2.4.25/setup.cfg	Fri May  6 13:31:14 2016
@@ -1,6 +1,6 @@
 [_ldap]
-library_dirs = /usr/lib /usr/lib64 /usr/local/lib /usr/local/lib64
-include_dirs = /usr/include /usr/include/sasl /usr/local/include /usr/local/include/sasl
+library_dirs =
+include_dirs = /usr/include/openldap /usr/include /usr/include/sasl /usr/local/include /usr/local/include/sasl
 defines = HAVE_SASL HAVE_TLS HAVE_LIBLDAP_R
 extra_compile_args = 
 extra_objects = 
